test/cpp/jit/test_constant_propagation.cpp

```diff
@@ -72,7 +72,7 @@ graph():
     RegisterOperators reg({
         Operator(
             "prim::test_tuple() -> (float[])",
-            [](const Node* node) {
+            [](const Node* node) -> Operation {
```

Comment:
Lots of these look like they can also be turned into the Operation form. Only a few are actually capturing state from the node.

Reply:
Yeah, there's definitely more improvements to be done here.  Is it alright if I do those in separate changes?

Reply:
Also, I'm a little surprised the explicit return type was necessary, but clang wouldn't accept the change without it.  Any ideas on if there's a way around it?